core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/migration/MultiregionalBucketMigrationActor.scala

```diff
@@ -702,6 +706,145 @@ object MultiregionalBucketMigrationActor {
       }
     } yield transferJob
 
+  final def refreshTransferJobs: MigrateAction[MultiregionalStorageTransferJob] =
+    for {
+      transferJob <- peekTransferJob
+      (storageTransferService, googleProject) <- asks { env =>
+        (env.storageTransferService, env.googleProjectToBill)
+      }
+
+      // Transfer operations are listed after they've been started. For bucket-to-bucket transfers
+      // we expect at most one operation. Polling the latest operation will allow for jobs to be
+      // restarted in the cloud console.
+      outcome <- liftF {
+        import OptionT.{fromOption, liftF}
+        for {
+          job <- liftF(storageTransferService.getTransferJob(transferJob.jobName, googleProject))
+          operationName <- fromOption[IO](Option(job.getLatestOperationName))
+          if !operationName.isBlank
+          operation <- liftF(storageTransferService.getTransferOperation(OperationName(operationName)))
+          outcome <- getOperationOutcome(operation)
+        } yield outcome
+      }
+
+      (status, message) = toTuple(outcome)
+      now <- nowTimestamp
+      _ <- inTransaction { _ =>
+        storageTransferJobs
+          .filter(_.id === transferJob.id)
+          .map(row => (row.finished, row.outcome, row.message))
+          .update(now.some, status.some, message)
+      }
+
+      _ <- getLogger[MigrateAction].info(
+        stringify(
+          "migrationId" -> transferJob.migrationId,
+          "transferJob" -> transferJob.jobName.value,
+          "finished" -> now,
+          "outcome" -> outcome.toJson.compactPrint
+        )
+      )
+
+    } yield transferJob.copy(finished = now.some, outcome = outcome.some)
+
+  final def getOperationOutcome(operation: TransferOperation): OptionT[IO, Outcome] =
+    OptionT {
+      Some(operation)
+        .filter(_.hasEndTime)
+        .traverse(_.getStatus match {
+          case TransferOperation.Status.SUCCESS => IO.pure(Success)
+          case TransferOperation.Status.ABORTED =>
+            IO.pure {
+              Failure(
+                s"""Transfer job "${operation.getTransferJobName}" failed: """ +
+                  s"""operation "${operation.getName}" was aborted."""
+              )
+            }
+          case TransferOperation.Status.FAILED =>
+            IO.pure {
+              Failure(
+                s"""Transfer job "${operation.getTransferJobName}" failed: """ +
+                  s"""operation "${operation.getName}" failed with errors: """ +
+                  operation.getErrorBreakdownsList
+              )
+            }
+          case other =>
+            // shouldn't really happen, but something could have gone wrong in the wire.
+            // report the error and try again later. todo: don't try indefinitely
```

Comment:
In this case, I think manual intervention is probably a reasonable way to stop an indefinite retry.

Looking through the status enum, the most likely to occur is probably `QUEUED`, in which case this would be the correct action.